python/lsst/obs/lsstSim/processEimage.py

```diff
@@ -25,7 +25,9 @@
 from lsst.pipe.tasks.processCcd import ProcessCcdTask
 from .eimageIsr import EimageIsrTask
 
+
 class ProcessEimageConfig(ProcessCcdTask.ConfigClass):
+
```

Comment:
I agree it looks funny, but unfortunately it is correct PEP8 and is what autopep8 did to the code. The rules are (at least approximately):
- A blank line is required after the class declaration before the doc string. This looks wrong to me, but it's PEP8.
- A blank line is required before each method definition. This is why the class doc string often has a blank line after it (not because the doc string must be followed by a blank line).